### ISSUE TITLE
Long-clicking a rippled button and then hiding it crashes MDC

### ISSUE THREAD

**Issue by robcaldecottvelo** [jamesmfriedman/rmwc#298]
This might be an MDC issue but it's serious so I thought I'd log it here too. Long-clicking on a ripple-enabled button/FAB and then immediately hiding it will cause an MDC crash.

Here is an example. Long click on the button (so you get a full ripple and then release the mouse button):

https://codesandbox.io/s/qzwm42p1vw

The code is:

```js
class App extends React.Component {
  state = { show: true };

  render() {
    return (
      <React.Fragment>
        <p>Long click on the button below (so you get a full ripple)</p>
        {this.state.show && (
          <Button
            raised
            onClick={() => {
              this.setState({ show: false });
            }}
          >
            Long click me
          </Button>
        )}
      </React.Fragment>
    );
  }
}
```

The crash occurs in this MDC code (`mdc.ripple.js`)

```js
removeClass: function removeClass(className) {
  return instance.root_.classList.remove(className);
}
```

The issue is that `root_` is `undefined` as the button is no longer being rendered.

Any advice is welcome. I will log this on the MDC issues page but if anyone can think of a workaround then I'm all ears.

**Comment by jamesmfriedman** [comment]
I updated the code sandbox in your comment. I've been moving some pieces around for a 2.0 alpha and just need to get it working again.

There are many places like this in the MDC api that assume the DOMnode will always be there, I'll hack around it.

**Comment by jamesmfriedman** [comment]
Fixed this with 2.0.0. Don't let the major version scare you, read the updated installation guide.
